### PR TITLE
[Ready for Review] Collab cafe notes - Jan-Apr 2024

### PR DESCRIPTION
**Pull request by aleesteele**
### Summary

Redo of #3487 - adding collab cafe notes from january, february, march, april 2024

Importantly, I removed the 'check-in' question answers for folks, after speaking with our Data Protection Officer at the Turing!

### CODE REVIEW

**Review by KirstieJane — COMMENTED** [review]
Thanks for archiving @aleesteele !

I've left a few formatting comments and I have an open question about why we're dropping the check-in questions. I'd prefer to keep them as they demonstrate the creativity and diversity in our attendees!

Might be something we have to fix forwards by drawing attendees attention to the point about the notes being archived in github and encouraging them to use the 🤫 emoji if there are answers they don't want to be stored here.

**Comment by aleesteele** [comment]
Thanks so much for the notes & review @KirstieJane!

Haven't forgotten about this! I'm speaking with Kit tomorrow at his drop-in hours to do a final review on the check-in question vs name for DP. We're getting this policy, which I'll make sure to drop in #3534 as well. Then I can get these notes updated and pushed for another review before we merge 😄

File: governance/community-calls/20240103-collaboration-cafe.md
Comment (aleesteele) [reply governance/community-calls/20240103-collaboration-cafe.md]: Weird - yep, fixing this formatting

File: governance/community-calls/20240221-collaboration-cafe.md
Comment (aleesteele) [reply governance/community-calls/20240221-collaboration-cafe.md:32]: Thanks @KirstieJane! I think I've been able to fix them – let me know if they're not rendering for you 😄

**Comment by aleesteele** [comment]
After one final review (where I found a few last rendering errors 😮‍💨 ), these are finally ready for another review! 

Updates on the names and archiving: After talking with our Data Protection officer, I re-added the names & check-in questions, and will be making to flag that more actively in future calls. Thanks for flagging Kirstie!

**Review by KirstieJane — APPROVED** [review]
Reviewed! Didn't look for typos but did look for text that shouldn't be shared on GitHub and didn't see anything 😸 

Thanks @aleesteele

**Pull request merged**
